services/spar/test-integration/Test/Spar/APISpec.hs

```diff
@@ -911,7 +913,84 @@ specAux = do
 
         sequence_ [ check tryowner perms | tryowner <- [minBound..], perms <- [0.. (length permses - 1)] ]
 
+specSSOSettings :: SpecWith TestEnv
+specSSOSettings = do
```

Comment:
A test which sends an empty json object `{}` to the PUT endpoint is missing. According to the comments, this should be forbidden.

Reply:
I check this for the JSON instance in the unit tests (services/spar/test/Test/Spar/APISpec.hs), but I can add it to the integration tests as well.